samples/snippets/create_secret_with_user_managed_replication.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+command line application and sample code for creating a new secret with
+user managed replication.
+"""
+
+import argparse
+
+
+def create_ummr_secret(project_id, secret_id, locations):
```

Comment:
```suggestion
# [START secretmanager_create_ummr_secret]
def create_ummr_secret(project_id, secret_id, locations):
```

Reply:
Please add start and end region tags for samples. See example here https://github.com/googleapis/python-secret-manager/blob/main/samples/snippets/create_secret.py#L22

Reply:
We do not intend to use this snippet in our docs. Just the sample had to added here. Thats why no need for region tags for now.